frontend_tests/zjsunit/namespace.js

```diff
@@ -1,4 +1,5 @@
 const _ = require('underscore/underscore.js');
+const proxyquire =  require('proxyquire').noCallThru();
```

Comment:
I think we also want [`.noPreserveCache()`](https://www.npmjs.com/package/proxyquire#forcing-proxyquire-to-reload-modules).

Reply:
Actually, going back to my notes from the slowly ongoing ES6 module migration, I think we want to consider [`rewiremock`](https://www.npmjs.com/package/rewiremock) instead of `proxyquire`.

Reply:
@andersk Okay. I will migrate it to `proxyquire`

Reply:
@andersk Okay. I will migrate it to `rewiremock `.